app/app.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra/base'
+require './lib/database_connection_setup'
+require './lib/peep'
+require './lib/user'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    redirect :'/peeps'
+  end
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/all'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new'
+  end
+
+  post '/peeps' do
+    user_id = session[:user_id]
+    Peep.create(text: params['text'], user_id: user_id)
+    redirect '/peeps'
+  end
+
+  get '/user/new' do
+    erb :'user/new'
+  end
+
+  post '/user' do
+    @user = User.create(name: params['name'],
+              username: params['username'],
+              email: params['email'],
+              password: params['password'])
+    session[:user_id] = @user.id
```

Comment:
we discussed that this could redirect to a specific user url.  At the moment you need a user id but users who aren't signed in have access to the peep form without getting an error. 👍